src/sonarqube-gui-plugin/src/main/resources/static/js/dashboard/main.js

```diff
@@ -206,8 +206,18 @@ SM.dashboard.loadCloneWidget = function(lang, key) {
 SM.dashboard.reInit = function() { // for fast debug/test reasons
   // clear view
   SM.pageBuilder.dashboard.build();
+  SM.dashboard.fetch();
+};
 
-  // clear cached state
+SM.dashboard.buildPage = function() {
+  SM.pageBuilder.dashboard.build();
+}
+/**
+ * Fetch/refresh data from the back end.
+ * @return {void}
```

Comment:
Please add a new line before `@return` and write `undefined` instead of void like before.